### ISSUE TITLE
Code breaks on latest release

### ISSUE THREAD

**Issue by HarryKodden** [wtforms/wtforms#581]
### Actual Behavior

```python

from wtforms.widgets import HTMLString
ImportError: cannot import name 'HTMLString'

```
### Environment
* Python version: 3.6.8
* wtforms version: 2.3.0

**Comment by spookey** [comment]
I had the same issue when updating.
``HTMLString`` was replaced with ``Markup``. So use ``from markupsafe import Markup`` instead.

Also ``from wtforms.widgets.core import escape_html`` was replaced by ``from markupsafe import escape``

**Comment by R11baka** [comment]
I had the same issue when running airflow

**Comment by hatched-DavidMichon** [comment]
Also got this issue

**Comment by kaxil** [comment]
Is it possible to restore `HTMLString` and cut `2.3.1` since it specifically said it was a backwards incompatible change in the [#400](https://github.com/wtforms/wtforms/pull/400)?

**Comment by davidism** [comment]
Fixed in #583.
